pages/basics/index.js

```diff
@@ -10,6 +12,15 @@ function Basics({url, content, locale}) {
       locale={locale}
       contributors={content.contributors}
     >
+      <Head>
+        {twitterCard({
+          card: 'summary',
+          title: content.title,
+          description: content.heading,
```

Comment:
We get line breaks in the `description` when it renders. Not sure if that affects the way it displays on Twitter. :thinking: